### ISSUE TITLE
Georeference Source vs Georeference Protocol

### ISSUE THREAD

**Issue by campmlc** [ArctosDB/arctos#3301]
I would like clarification on how to use Georeference Source vs Georeference Protocol when coordinates were determined after the fact for legacy data, using GeoLocate. Should both fields be "GeoLocate"? 
Also, we do not have a remarks field for georeferencing information. Should any remarks from the person who did the georeferencing go under specimen event remarks? This should be clear in the documentation.

**Comment by campmlc** [comment]
This also brings up the issue of "georeferencing agent". We assume the agent is the same as the event determiner. But that should be made explicit. Perhaps we need another Georeferencing Source-like field to mean "curatorial determination" vs "collector determination".

**Comment by mkoo** [comment]
When using Geolocate, it's a bit confusing since we tend to consider it as
both the source and its own protocol.
But in general:
Georeference Source - references used for the geocoordinates determination,
eg., Google Maps v2, USGS Quad {name here} 2010,

Georeference Protocol - MaNIS including error estimates from the error
calculator, etc or commonly 'not recorded' if no errors included from
legacy data
If we have GPS coordinates (and no error or other spatial info) from the
collector we usually use:
 a default Max Error of 30; Georeference Source = GPS unit; Georeference
Protocol = MaNIS (since guidelines use 30 m as default for this situation)

If it's from the collector, then we use their name in the specimen event as
determiner. it's not assumed for us.

I dont think we need more free-text fields here. Use locality remarks for
georeferencing assumptions and comments, use specimen event remarks for
things like field note annotations and other determination issues.

Remember that the archive/history link will show you who was logged in and
making changes last.

Need to tag for updating the documentation in Handbook if it's not more
clear there! (and probably more exmaples wouldnt hurt!)

**Comment by tucotuco** [comment]
The [Georeferencing Quick Reference Guide](https://doi.org/10.35035/e09p-h128) is going to be released before the end of the year. We recommend that as a protocol to reference (if it is used) and the [Georeferencing Calculator](http://georeferencing.org/georefcalculator/gc.html) (updated and with [new manual](https://doi.org/10.35035/gdwq-3v93)) has it as one of the protocol options to select ("Georeferencing Quick Reference Guide. 2020"). The [Georeferencing Best Practices](https://doi.org/10.15468/doc-gg7h-s853), also being released with the other two, still recommends 30 m for GPS readings after 2020-05-01 that have no recorded accuracy from the apparatus.

**Comment by Jegelewicz** [comment]
> Need to tag for updating the documentation in Handbook if it's not more clear there! (and probably more exmaples wouldnt hurt!)

I updated the documentation to change from "reference source" to "Georeference Source" - https://handbook.arctosdb.org/documentation/locality.html#georeference-source

I think it seems pretty clear, but if it isn't, please either edit or let me know what could make it better.

**Comment by dustymc** [comment]
>updated the documentation 

This breaks links. Please at least update

<img width="173" alt="Screen Shot 2020-12-18 at 10 47 01 AM" src="https://user-images.githubusercontent.com/5720791/102649640-61c42180-411e-11eb-9678-f149fb31187d.png">

if something in the documentation absolutely must be changed. I got that one.

https://handbook.arctosdb.org/how_to/How-to-Contribute-Content-to-Arctos-Handbook.html#titles

**Comment by Jegelewicz** [comment]
A quick aside on the above. Is this where all of the stuff comes from when you click the "help links"? Do we have some documentation about how all of those blue words actually work? Here is an example of where we are failing to get people to the help they might need:

depth - on the edit locality page, click either min depth or max depth and you are rewarded with:

![image](https://user-images.githubusercontent.com/5725767/102651935-99d16180-412a-11eb-8121-978017db3645.png)

Yet, we do have some documentation - https://handbook.arctosdb.org/documentation/locality.html#depths

and clearly there is stuff in this table:
![image](https://user-images.githubusercontent.com/5725767/102652975-81ae1200-412b-11eb-80a8-2157ee7c3e57.png)

I've been messing around for a half hour and I could not figure out how this system functions, although after this I think I have an idea now. Maybe?

Is the failure to get the depth help because this:

```
<label for="min_depth" class="helpLink" id="_depth">Min. Depth</label>
```

should really be this

```
<label for="min_depth" class="helpLink" id="_min_depth">Min. Depth</label>
```

or do these two things have absolutely nothing to do with each other?

I feel like this is a really complicated thing that should be much easier....but that could just be my inexperience!

**Comment by dustymc** [comment]
> where all of the stuff comes from when you click the "help links"? 

Yes, but it also builds components (like specimenresults) dynamically - the warnings are real.

>should really be this

Yup. (``$(document).on("click", ".helpLink", function(e) {..`` in ajax.js if you want details)

>really complicated 

Yea, there's some complexity involved - I think it's elegantly simple for what it does (and can do) but I'm biased.

**Comment by Jegelewicz** [comment]
It probably is elegantly simple, but we need some good documentation for how it works otherwise it will be busted a lot.

Also - it is super annoying to have to click through the documentation pop-up to get to the good stuff....how can that be avoided?

**Comment by dustymc** [comment]
>how can that be avoided?

It's in the good documentation at the top of the form....

**Comment by Jegelewicz** [comment]
Don't you know that nobody reads anything in a red box any more?

OK - I'll try to make better use of this.
